src/test/java/org/medibloc/panacea/GrpcAolTest.java

```diff
@@ -133,12 +135,31 @@ private void testAddRecord(MsgAddRecord msg) throws PanaceaApiException, IOExcep
                 msg,
                 memo,
                 fee,
-                BroadcastMode.BROADCAST_MODE_BLOCK);
+                BroadcastMode.BROADCAST_MODE_SYNC);  // Used the sync mode as an example
 
         TxResponse response = client.broadcast(request);
         System.out.println(response.toString());
         Assert.assertNotNull(response.getTxhash());
         Assert.assertEquals(0, response.getCode());
+
+        String txHash = response.getTxhash();
+
+        while (true) {
+            try {
+                response = client.getTxResponse(txHash);
+
+                TxMsgData txMsgData = TxMsgData.parseFrom(Hex.decodeHex(response.getData()));
+                MsgAddRecordResponse msgRes = MsgAddRecordResponse.parseFrom(txMsgData.getData(0).getData());
+                System.out.println("offset: " + msgRes.getOffset());
```

Comment:
I think we need logic to verify it.

```suggestion
                Assert.assertEquals(0, msgRes.getOffset());
                System.out.println("offset: " + msgRes.getOffset());
```

Reply:
Yeah. I added a new argument `expectedOffset` to the `testAddRecord()` function, so that the function can be used for any case.
b6f7f8f59904bc42d2eb83e56bd2f02ec6079026

Reply:
Yeah. I added a new argument `expectedOffset` to the `testAddRecord()` function, so that the function can be used for any case.
b6f7f8f59904bc42d2eb83e56bd2f02ec6079026

Reply:
Good. Thank you. 👍